main.rb

```diff
@@ -0,0 +1,12 @@
+require './users.rb'
+require './database_setup.rb'
+
+
+# setup_database()
+
+
+
+u1=User.new('chinku','chinku@gmail.com','password','user')
+u1.save()
```

Comment:
if you are not passing anything as args to a method, you can avoid `()` while calling it in ruby